pwa/src/styling/main.css

```diff
@@ -0,0 +1,9 @@
+.utrecht-page {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
+
+.utrecht-page-content {
+    flex-grow: 1;
+}
```

Comment:
Let's try not to call the actual classNames, instead make our own classNames. E.g. instead of `utrecht-page` we create `Page`.